docs/amber_script.md

```diff
@@ -394,17 +408,23 @@ BUFFER vertex kData TYPE vec3<int32> DATA
  1  1  0xff800080
 END
 
-BUFFER index kIndices TYPE int32 DATA
+BUFFER kIndices TYPE int32 DATA
 0  1  2    2  1  3
 4  5  6    6  5  7
 8  9  10   10 9  11
 12 13 14   14 13 15
 END
 
+PIPELINE graphics kGraphicsPipeline
+  ATTACH kVertexShader
+  ATTACH kFragmentShader
+
+  VERTEX_DATA kData
+  INDEX_DATA kIndices
+END  # pipeline
+
 CLEAR_COLOR kGraphicsPipeline 255 0 0 255
```

Comment:
Now that FRAMEBUFFER is not declared in a pipeline, the CLEAR and CLEAR_COLOR have to name an buffer rather than a pipeline, I think.

Reply:
Well, you're still clearing the drawing buffer for the pipeline aren't you as opposed to some other buffer?

Reply:
Let's resolve Framebuffer semantics in a future CL.